runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesPlugin.java

```diff
@@ -62,4 +98,75 @@ protected void shutDown() throws Exception
 	{
 		overlayManager.remove(overlay);
 	}
+
+	@Subscribe
+	public void onWidgetLoaded(final WidgetLoaded event)
+	{
+		if (itemPricesConfig.showHighAlchOnGuidePrices())
+		{
+			if (WidgetID.GUIDE_PRICES_GROUP_ID == event.getGroupId())
+			{
+
+				final Widget parentWidget = client.getWidget(WidgetInfo.GUIDE_PRICES_MAIN_CONTAINER);
+
+				final Widget totalGePricesWidget = client.getWidget(WidgetInfo.GUIDE_PRICES_ITEMS_VALUE);
+				totalGePricesWidget.setWidthMode(WidgetSizeMode.ABSOLUTE);
+				totalGePricesWidget.setOriginalWidth(totalGePricesWidget.getWidth() / 2);
+				totalGePricesWidget.revalidate();
+
+				this.totalHighAlchValueWidget = parentWidget.createChild(-1, WidgetType.TEXT);
+				totalHighAlchValueWidget.setFontId(FontID.PLAIN_12);
+				totalHighAlchValueWidget.setTextShadowed(true);
+				totalHighAlchValueWidget.setTextColor(0xFF981F);
+				totalHighAlchValueWidget.setXTextAlignment(1);
+				totalHighAlchValueWidget.setYTextAlignment(1);
+				totalHighAlchValueWidget.setText(createHighAlchemyValueText(0L));
+				totalHighAlchValueWidget.setOriginalWidth(totalGePricesWidget.getWidth());
+				totalHighAlchValueWidget.setOriginalHeight(totalGePricesWidget.getHeight());
+				totalHighAlchValueWidget.setOriginalX(totalGePricesWidget.getOriginalX() + totalGePricesWidget.getWidth());
+				totalHighAlchValueWidget.setOriginalY(totalGePricesWidget.getOriginalY());
+			}
+		}
+	}
+
+	@Subscribe
+	public void onItemContainerChanged(final ItemContainerChanged event)
+	{
+		if (itemPricesConfig.showHighAlchOnGuidePrices())
+		{
+			if (InventoryID.GUIDE_PRICES.getId() == event.getContainerId())
+			{
+				final ItemContainer itemContainer = event.getItemContainer();
+				final Item[] items = itemContainer.getItems();
+				final long totalHighAlchValue = calculateTotalHighAlchValue(items);
```

Comment:
Does this have a use case for `net.runelite.client.plugins.bank.ContainerCalculation` ?

Reply:
I want to use that but this is package class and I don't know if plugins can interact with each other. If this is not a problem I can make this class public or move it to another where other plugins can use it.